docker-compose-native/docker-compose.yml

```diff
@@ -66,21 +75,23 @@ services:
     networks:
       presto:
         ipv4_address: 172.19.0.4
+
   worker_1:
     container_name: worker_1
-    image: public.ecr.aws/oss-presto/presto-native:0.286-20231205020058-ada5eb3-amd64
-    platform: linux/amd64
+    image: public.ecr.aws/oss-presto/presto-native:0.289-ubuntu-arm64
+    platform: linux/arm64
     restart: always
     deploy:
       resources:
         limits:
           memory: 2G
+    ports:
+      - 8081:8081
```

Comment:
I think this should be
`- 8081:8080`

Reply:
Could you please share why? It was updated to match the port from config `http-server.http.port`, which is used by the prestissimo worker.